schema/new_library_verification_cases_schema.json

```diff
@@ -0,0 +1,3604 @@
+                            },
+                            "required": [
+                                "idf",
+                                "idd",
+                                "weather",
+                                "output",
+                                "ep_path"
+                            ]
+                        },
+                        "expected_result": {
+                            "type": "string"
+                        },
+                        "datapoints_source": {
+                            "type": "object",
+                            "properties": {
+                                "idf_output_variables": {
+                                    "type": "object",
+                                    "properties": {
+                                        "T_sa_set": {
+                                            "type": "object",
+                                            "properties": {
+                                                "subject": {
+                                                    "type": "string"
+                                                },
+                                                "variable": {
+                                                    "type": "string"
+                                                },
+                                                "frequency": {
+                                                    "type": "string"
+                                                }
+                                            },
+                                            "required": [
+                                                "subject",
+                                                "variable",
+                                                "frequency"
+                                            ]
+                                        }
+                                    },
+                                    "required": [
+                                        "T_sa_set"
+                                    ]
+                                },
+                                "parameters": {
+                                    "type": "object",
+                                    "properties": {
+                                        "T_z_coo": {
+                                            "type": "number"
+                                        }
+                                    },
+                                    "required": [
+                                        "T_z_coo"
+                                    ]
+                                }
+                            },
+                            "required": [
+                                "idf_output_variables",
+                                "parameters"
+                            ]
+                        },
+                        "verification_class": {
+                            "type": "string"
+                        }
```

Comment:
For this, it would help to include "const": ""SupplyAirTempReset" to verify that datapoints_source matches the verification_class. This also applies to the other objects